### PR TITLE
Add support to Prometheus

### PR DESCRIPTION
**Pull request by HileQAQ**
Now we can provide a port (for example, in `/etc/overlaybd-snapshotter/config.json`, we set `monitorPort: 9099`) for prometheus to monitor some metrics like GRPC APIs latency or error count.

(from https://github.com/containerd/accelerated-container-image/issues/140)

Signed-off-by: Haoqi Miao <miaohaoqi.mhq@alibaba-inc.com>

### CODE REVIEW

File: cmd/overlaybd-snapshotter/main.go [hunk 1]
```diff
@@ -64,6 +64,10 @@ func main() {
 	}
 	logrus.Infof("%+v", *pconfig)
 
+	if pconfig.MonitorPort != "" {
```
Comment (BigVan) [inline cmd/overlaybd-snapshotter/main.go:67]: Add MonitorPort in config.json ?

**Comment by liulanzheng** [comment]
@HileQAQ  remove this binary? cmd/overlaybd-snapshotter/overlaybd-snapshotter

File: docs/PROMETHEUS.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+```json
+{
+    "root": "/var/lib/containerd/io.containerd.snapshotter.v1.overlaybd",
+    "address": "/run/overlaybd-snapshotter/overlaybd.sock",
+    "verbose": "info",
+    "rwMode": "overlayfs",
+    "logReportCaller": false,
+    "autoRemoveDev": false,
+    "exporterConfig": {
+        "enable": false,
```
Comment (liulanzheng) [inline docs/PROMETHEUS.md:18]: true?

**Review by liulanzheng — APPROVED** [review]
LGTM

**Pull request merged**